.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+	"env": {
+		"es6": true,
+		"jest": true,
```

Comment:
We can also add [eslint-plugin-jest](https://www.npmjs.com/package/eslint-plugin-jest) setup. You can check here, how I did it for Gutenberg: https://github.com/WordPress/gutenberg/pull/1382/files#diff-df39304d828831c44a2b9f38cd45289cR8.